src/types.ts

```diff
@@ -44,6 +44,6 @@ export interface Pagination {
 }
 
 export interface PaginatedResponse<T> {
-    pagination?: Pagination;
     [key: string]: T[] | Pagination | undefined;
+    pagination?: Pagination;
```

Comment:
Thank you for properly sorting 👍

Reply:
that was one of the failing lints. I agree much cleaner.